src/pages/Profile/Profile.jsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-// import person from '../../images/person.png';
-// ^^ blank avatar for profile pic, uncomment the above line to use
+import person from '../../images/person.png';
 import './Profile.css';
 
 const Profile = ({ firstName, lastName, stats }) => {
   return (
     <div className="profile-page">
-      {/* TODO:
-      Complete the profile page here.
-      For the profile picture, use the avatar placeholder image that has
-      already been imported for you on line 3.
-      For the name & stats to display, use the props (firstName, lastName, and stats)
-      that have already been declared. */}
-      {firstName} {lastName} {stats}
+      <div className="avatar">
+        <img src={person} />
+      </div>
+      <div>
+        <div id="name">{firstName} {lastName}</div>
```

Comment:
on my end, the name wraps to the next line, making the text overlap with the stats. 
![image](https://user-images.githubusercontent.com/67395310/199406205-5f10c7d4-73da-4f73-8ad3-364cad1057fd.png)